ktesting/testinglogger.go

```diff
@@ -62,6 +63,8 @@ import (
 type TL interface {
 	Helper()
 	Log(args ...interface{})
+	Cleanup(func())
+	Name() string
```

Comment:
This is an API break (see https://pkg.go.dev/golang.org/x/exp/apidiff#section-readme, "Interfaces").

We could eliminate this interface and instead use testing.TB. But we want to allow custom implementations of the interface, not the least in our own NopTL, so I think we should keep this as-is and hopefully won't need additional methods in the future.